native/packages/react-native-backpack/android/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'Backpack2'
```

Comment:
We should have a TODO here to change this

Reply:
done